src/app/modules/user/user.controller.ts

```diff
@@ -215,50 +209,38 @@ export const handleFetchUser: ControllerHandler = async (req, res) => {
     })
 }
 
-export const _handleUpdateUserLastSeenFeatureUpdateDate: ControllerHandler<
-  unknown,
-  string,
-  UpdateUserLastFeatureUpdateDateDto
-> = async (req, res) => {
-  const { latestLastSeenFeatureUpdateDate } = req.body
-  const sessionUserId = getUserIdFromSession(req.session)
-
-  if (!sessionUserId) {
-    return res.status(StatusCodes.UNAUTHORIZED).json(UNAUTHORIZED_USER_MESSAGE)
-  }
-
-  return updateUserLastSeenFeatureUpdateDate(
-    sessionUserId,
-    latestLastSeenFeatureUpdateDate,
-  )
-    .map(() => {
-      return res
-        .status(StatusCodes.OK)
-        .json(UPDATE_LAST_SEEN_FEATURE_DATE.SUCCESS)
-    })
-    .mapErr((error) => {
-      logger.error({
-        message: UPDATE_LAST_SEEN_FEATURE_DATE.ERROR,
-        meta: {
-          action: 'handleUpdateUserLastSeenFeatureUpdate',
-          userId: sessionUserId,
-        },
-        error,
-      })
-
-      const { errorMessage, statusCode } = mapRouteError(error)
-      return res.status(statusCode).json(errorMessage)
-    })
-}
-
 /**
- * @route POST /whats-new/last-seen
+ * @route POST /flag/last-seen
  * @returns 200 when user last seen feature update date update success
  * @returns 401 if user is not currently logged in
  * @returns 422 when user id does not exist in the database
  * @returns 500 database errors occurs
  */
-export const handleUpdateUserLastSeenFeatureUpdateDate = [
-  validateUpdateUserLastSeenFeatureUpdateParams,
-  _handleUpdateUserLastSeenFeatureUpdateDate,
-] as ControllerHandler[]
+export const handleUpdateUserLastSeenFeatureUpdateDate: ControllerHandler =
+  async (req, res) => {
+    const currentDateTime = new Date(Date.now())
```

Comment:
should this just be `new Date()`? i think its the same thing

Reply:
Yeap they are the same thing. I will make the appropriate change